pkg/controller/operator/common/defaults.go

```diff
@@ -838,12 +838,6 @@ items:
     name: pod-security-policy
     labels:
       addons.kubermatic.io/ensure: true
-- apiVersion: kubermatic.k8s.io/v1
-  kind: Addon
-  metadata:
-    name: logrotate
-    labels:
-      addons.kubermatic.io/ensure: true
```

Comment:
It appears that removing this should also result to data migration (in kubermatic-installer).

Reply:
I created a [PR](https://github.com/kubermatic/kubermatic-installer/pull/137) in kubermatic-installer, even if I'm not sure about it. Isn't it supposed to be used for kubermatic < 2.14?

Reply:
Sorry, I meant this kubermatic-installer https://github.com/kubermatic/kubermatic/tree/master/cmd/kubermatic-installer

P.S.
We should get rid of that one in dedicated repo.

Reply:
ok, gonna close the PR then :)

Reply:
May I add data migration in this PR or does it worth a separate PR?

Reply:
yeah, in here would be perfect